src/container/srv_target.c

```diff
@@ -861,29 +864,22 @@ cont_child_stop(struct ds_cont_child *cont_child)
 		cont_close_hdl(hdl->sch_uuid);
 	}
 
-	/* Some ds_cont_child will only created by ds_cont_child_lookup().
-	 * never be started at all
-	 */
-	cont_child->sc_stopping = 1;
-
 	/* Stop DTX reindex by force. */
 	stop_dtx_reindex_ult(cont_child, true);
 
-	if (cont_child_started(cont_child)) {
```

Comment:
Consider `ds_cont_local_open()` for rebuild, the cleanup logic may call `cont_child_stop()` with `cont_child_started()` as false. So here, removing the condition check maybe unsafe?

Reply:
As long as cont_child_stop() is always called for an started cont, and cont_child_stop() won't be called twice for an started cont, I think we should be fine.

@Nasf-Fan , does your example break above two consumptions?

Reply:
> As long as cont_child_stop() is always called for an started cont,
Here, we removed the condition check, then it is the caller to guarantee that.